provd/main.py

```diff
@@ -187,35 +187,30 @@ def startService(self) -> None:
         Service.startService(self)
 
 
-class HTTPProcessService(Service):
-    def __init__(
-        self,
-        prov_service: ProvisioningService,
-        process_service: ProcessService,
-        config: ProvdConfigDict,
-    ) -> None:
```

Comment:
I see that this PR removes some type hints. Was that intentional?

Reply:
Definitely not, probably just a wrong merge resolution, thanks!